sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -89,8 +89,8 @@
     @Parameter(names = "-readWriteQueryPct", description = "Comma separated read write query workload percent")
     private String readWriteQueryPct = "90,9,1";
 
-    @Parameter(names = "-manageResources", description = "Control switch for creating/deleting underlying test resources")
-    private boolean manageResources = false;
+    @Parameter(names = "-manageDatabase", description = "Control switch for creating/deleting underlying database resource")
```

Comment:
This uses an existing config.
did `manageResources` only control database or container as well?
doesn't this cause any problem for CTL, etc to limit it to database? @simplynaveen20

Reply:
manageResources controlled the container creation before. In the container, it was always set to true. Instead now
1. Collection/container creation happens by default
2. The Database creation is an opt-in feature

Thanks @moderakh, as always, for the quick review.